### PR TITLE
[WIP] Restores ie11 support

### LINKED ISSUE #547: App is a blank white screen in IE11
We believe this is due to the lack of support for AddEventListener and arrow functions in: https://github.com/pulibrary/approvals/blob/master/app/assets/javascripts/no_flash.js

This was put in place as a fix to FOUC that users were seeing, but the actual problem was json data being drawn to the screen before LUX components could render. The issue was solved by introducing pagination, so the `no_flash.js` script referenced above should be safe to remove. Should be tested in IE11 in BrowserStack.

### PR DESCRIPTION
**Pull request by sdellis**
Fixes #547 

Removes the JS-dependent fix for FOUC and replaces with[ a CSS-only approach](https://gist.github.com/electrotype/7960ddcc44bc4aea07a35603d1c41cb0).

Fixes other IE11 code/display issues as well.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -26,12 +26,17 @@
...
- 
- .js {
+
+.js {
   visibility: visible;
   opacity: 1;
- }
+}
+
+.form-button {
```
Comment (carolyncole) [inline app/assets/stylesheets/application.scss]: @sdellis What is this for?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
-<html class="no-js">
+<html>
   <head>
     <title>Leave and Travel Requests</title>
+    <style>html{visibility: hidden;opacity:0;}</style>
```
Comment (carolyncole) [inline app/views/layouts/application.html.erb]: Will adding the style here deal with the browser caching the page and the flash return

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -25,7 +26,7 @@
           {name: 'Help', component: 'Help', href: '/help'},
           {name: 'Logout', component: 'Logout', href: '/sign_out'}
         <% end %>
-      ]"/>
+      ]"></menu-bar>
```
Comment (carolyncole) [inline app/views/layouts/application.html.erb]: This seems to be solving something else

File: app/views/travel_requests/_title_card.html.erb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <card size="full-width" class="title-card">
   <card-media>
     <lux-icon-base width="50" height="50" icon-color="rgb(65, 70, 78)">
-      <<%= request.travel_category_icon %> />
+      <<%= request.travel_category_icon %>></<%= request.travel_category_icon %>>
```
Comment (carolyncole) [inline app/views/travel_requests/_title_card.html.erb]: Is this relates to IE support?

Comment (sdellis) [reply app/views/travel_requests/_title_card.html.erb]: Yes, this relates to IE11 support. IE11 complains when there are not full closing tags and it affects the layout of the page. Would it be better to split this into a fresh issue/PR?

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (sdellis) [reply app/views/layouts/application.html.erb]: According to the console errors, IE11 requires full closing tags. Only the header renders without this fix.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (sdellis) [reply app/assets/stylesheets/application.scss]: These are just indentation fixes, followed by a rule to show the page after all other css rules have loaded.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (sdellis) [reply app/views/layouts/application.html.erb]: I'm not sure I understand the problem you're asking about. I think this is more about the order in which the CSS is loaded. Does browser caching affect the order in which the CSS is loaded?

**Pull request merged**